### ISSUE TITLE
Labs Select - a11y violation

### ISSUE THREAD

**Issue by donovangini** [Workday/canvas-kit#761]
## 🐛 Bug Report

The `button` element in the Labs Select has an `aria-owns` that points to the popup. Should probably be `aria-controls` according to [this conversation](https://github.com/w3c/html-aria/issues/124)

## To Reproduce

Use Labs Select in code and run a test with a tool like `@workday/uic-common-utils-a11y`

## Expected Behavior

No a11y violations.

## Actual Results

An a11y violation.

### Error Output

Paste the error output here

```javascript
"violations": [
      {
        "id": "aria-valid-attr-value",
        "impact": "critical",
        "tags": [
          "cat.aria",
          "wcag2a",
          "wcag412"
        ],
        "description": "Ensures all ARIA attributes have valid values",
        "help": "ARIA attributes must conform to valid values",
        "helpUrl": "https://dequeuniversity.com/rules/worldspace/3.4/aria-valid-attr-value?application=attestAPI",
        "nodes": [
          {
            "any": [],
            "all": [
              {
                "id": "aria-valid-attr-value",
                "data": [
                  "aria-owns=\"a006582fc-44ed-46ee-aa0c-7b053cc2790b\""
                ],
                "relatedNodes": [],
                "impact": "critical",
                "message": "Invalid ARIA attribute value: aria-owns=\"a006582fc-44ed-46ee-aa0c-7b053cc2790b\""
              }
            ],
            "none": [],
            "impact": "critical",
            "html": "<button aria-haspopup=\"listbox\" aria-owns=\"a006582fc-44ed-46ee-aa0c-7b053cc2790b\" type=\"button\" value=\"English\" data-automation-id=\"language\" id=\"input-1\" class=\"css-10eneyg\">English</button>",
            "target": [
              "#input-1"
            ],
            "failureSummary": "Fix all of the following:\n  Invalid ARIA attribute value: aria-owns=\"a006582fc-44ed-46ee-aa0c-7b053cc2790b\""
          },
          {
            "any": [],
            "all": [
              {
                "id": "aria-valid-attr-value",
                "data": [
                  "aria-owns=\"ad3272ae5-21cd-43b2-84d6-93682a3dfc64\""
                ],
                "relatedNodes": [],
                "impact": "critical",
                "message": "Invalid ARIA attribute value: aria-owns=\"ad3272ae5-21cd-43b2-84d6-93682a3dfc64\""
              }
            ],
            "none": [],
            "impact": "critical",
            "html": "<button aria-haspopup=\"listbox\" aria-owns=\"ad3272ae5-21cd-43b2-84d6-93682a3dfc64\" type=\"button\" value=\"\" data-automation-id=\"disability\" id=\"input-9\" class=\"css-10eneyg\">WDRES.PROMPT.LABEL.SelectOne</button>",
            "target": [
              "#input-9"
            ],
            "failureSummary": "Fix all of the following:\n  Invalid ARIA attribute value: aria-owns=\"ad3272ae5-21cd-43b2-84d6-93682a3dfc64\""
          }
        ]
      }
    ]
```

**Comment by jamesfan** [comment]
Chatting through with @donovangini on this now. It doesn't seem like replacing `aria-owns` with `aria-controls` gets rid of the violation (even though it may still be the right thing to do). Digging further.

**Comment by jamesfan** [comment]
Chatting through with @donovangini on this now. It doesn't seem like replacing `aria-owns` with `aria-controls` gets rid of the violation (even though it may still be the right thing to do). Digging further.

**Comment by jamesfan** [comment]
I think it might be related to the menu not existing until you interact with the button. Adjusting it so that `aria-owns` (or `aria-controls`) only exists if the menu exists seems to solve the issue.
